_posts/en/newsletters/2021-03-03-newsletter.md

```diff
@@ -103,7 +103,16 @@ BOLTs][bolts repo].*
 - [Rust-Lightning #791][] adds support for polling `BlockSource` interfaces on
   startup to sync blocks and headers and fork detection during sync.
 
-- [Rust-Lightning #794][] Add support for `opt_shutdown_anysegwit` feature #780 FIXME:jonatack
+- [Rust-Lightning #794][] enables support for the [BOLT2][]
+  `option_shutdown_anysegwit` feature that permits future segwit versions when
+  initiating shutdown. If `option_shutdown_anysegwit` is negotiated, a channel
+  party sending a shutdown message to initiate closing may send a scriptpubkey
+  for payment, provided the script complies with the standard [BIP141][] witness
+  program form of a *version byte* (a 1-byte push opcode of `OP_1` through
+  `OP_16`) followed by a *witness program* (a byte vector push of 2 to 40
+  bytes). These shutdown scripts are limited to standard forms to avoid
+  expensive fee-heavy scripts or transactions with oversized scripts not
+  propagating due to non-standardness.
```

Comment:
@ariard had a look, acked, and suggested perhaps referencing the lightning spec pull https://github.com/lightningnetwork/lightning-rfc/pull/672 of which PR 794 is an implementation. I'll leave the call up to reviewers and @harding ;)